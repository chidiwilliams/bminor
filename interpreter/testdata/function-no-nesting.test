square: function integer ( x: integer ) = {
    nested: function integer ( y: integer ) = {
    	return y^2;
    }

	return x^2;
}
