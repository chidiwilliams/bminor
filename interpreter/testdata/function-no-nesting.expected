err:Error on line 2: function definitions may not be nested
err: